src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

![image](https://user-images.githubusercontent.com/12864128/233040657-19c7bd88-0e9a-4bab-8e81-b953bfcff441.png)